source/testnet/guides/become-baker.rst

```diff
@@ -135,6 +110,19 @@ concept of an *epoch* as a unit of time that defines a period in which the set
 of current bakers and stakes are fixed. Epochs have a time duration fixed at the
 Genesis block. In the testnet, epochs have a duration of **1 hour**.
 
+When performing changes, once the transaction that performs the change is
+finalized, this change can be in two different states:
+
+- When a change is said to be **pending** it means the change is processed
+  and will eventually be executed.
```

Comment:
```suggestion
- A **pending** change has been processed and will eventually be executed.
```

Reply:
Section has been removed so this is no longer applicable.